.babelrc

```diff
@@ -0,0 +1,17 @@
+{
+  "presets": [
+    ["env", {
+      "modules": false,
+      "targets": {
+        "browsers": ["> 1%", "last 2 versions"],
+        "uglify": true
```

Comment:
https://babeljs.io/docs/plugins/preset-env/#targetsuglify notes:

> Note: This option is deprecated in 2.x and replaced with a forceAllTransforms option.

Reply:
This file was automatically generated by Webpacker, so I trust they added this on purpose. It looks like webpacker has locked the `babel-preset-env` dependency to less than 2.x, so this should be fine to leave as is.